packages/http/libraries/core/src/http/decorators/RequestParam.ts

```diff
@@ -16,14 +20,21 @@ export function requestParam(
     key: string | symbol | undefined,
     index: number,
   ): void => {
-    let parameterMetadataObject:
-      | {
-          [key: string]: ControllerMethodParameterMetadata[];
-        }
-      | undefined = getReflectMetadata(
-      target.constructor,
-      controllerMethodParameterMetadataReflectKey,
-    );
+    const controllerFunction: ControllerFunction | undefined = (
+      target as Controller
+    )[key as string];
```

Comment:
Let's avoid accesing if key is undefined:

```suggestion
    const controllerFunction: ControllerFunction | undefined = key === undefined ? undefined : (
      target as Controller
    )[key];
```